src/test/scripts/helm_install.sh

```diff
@@ -227,6 +227,30 @@ package_functest_helm_chart() {
   helm package "$FUNCTEST_CHART_PATH" --destination "$HELM_PACKAGE_DIR"
 }
 
+install_secrets() {
+    if [[ -n "${BAMBOO_LICENSE}" ]]; then
```

Comment:
Is this just for testing locally? We can install the secrets to our CI cluster if necessary.

Reply:
I mention this in the description above; I'm not sure if this is appropriate, I'm happy to remove it. However we should have somewhere where this is automated/documented as a prerequisite for the CI cluster.